src/libraries/System.Private.Xml/src/System/Xml/XmlResolver.ThrowingResolver.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    public abstract partial class XmlResolver
+    {
+        /// <summary>
+        /// Gets an <see cref="XmlResolver"/> which forbids entity resolution.
```

Comment:
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property says that property summaries should not link to the type.

```suggestion
        /// Gets an XML resolver which forbids entity resolution.
```